contributors/design-proposals/standardization-of-development-process.md

```diff
@@ -0,0 +1,356 @@
+
+as a project it is vital to be able to track the chain of custody for a proposed
+enhancement from conception through implementation. This proposal does not
+attempt to mandate how SIGs track their work internally, however, it is
+suggested that SIGs which do not adhere to a process which allows for their hard
+work to be explained to others in the wider Kubernetes community will see their
+work wallow in the shadows of obscurity. At the very least [survey data][]
+suggest that high quality documentation is crucial to project adoption.
+Documentation can take many forms and it is imperative to ensure that it is easy
+to produce high quality user or developer focused documentation for a complex
+project like Kubernetes.
+
+The use of GitHub issues when proposing changes does not provide SIGs good
+facilities for signaling approval or rejection of a proposed change to Kubernetes
+since anyone can open a GitHub issue at any time. Additionally managing a proposed
+change across multiple releases is somewhat cumbersome as labels and milestones
+need to be updated for every release that a change spans. These long lived GitHub
+issues lead to an ever increasing number of issues open against
+`kubernetes/features` which itself has become a management problem.
+
+In addition to the challenge of managing issues over time, searching for text
+within an issue can be challenging. The flat hierarchy of issues can also make
+navigation and categorization tricky. While not all community members might
+not be comfortable using Git directly, it is imperative that as a community we
+work to educate people on a standard set of tools so they can take their
+experience to other projects they may decide to work on in the future. While
+git is a fantastic version control system (VCS), it is not a project management
+tool nor a cogent way of managing an architectural catalog or backlog; this
+proposal is limited to motivating the creation of a standardized definition of
+work in order to facilitate project management. This primitive for describing
+a unit of work may also allow contributors to create their own personalized
+view of the state of the project while relying on Git and GitHub for consistency
+and durable storage.
+
+Ideally release notes should [tell a story][] which
+is compelling enough to encourage users and operators to upgrade their clusters.
+Without a standardized mechanism for describing important enhancements our
+talented technical writers and product managers struggle to weave a coherent
+narrative explaining why a particular release is important. Additionally for
+critical infrastructure such as Kubernetes adopters need a forward looking road map
+in order to plan their adoption strategy.
+
+[tell a story]: https://blog.rust-lang.org/2017/08/31/Rust-1.20.html 
+[road to Go 2]: https://blog.golang.org/toward-go2
+[survey data]: http://opensourcesurvey.org/2017/
+
+## Detailed design
+
+### What type of work should be tracked by a KEP
+
```

Comment:
I would suggest defining what KEP stands for before using this (presumed) acronym.